### DISCUSSION TITLE
Sound Adjust

### DISCUSSION THREAD

**Discussion by gnokketto** [harbaum/galagino#40] [Q&A]
Hi, Does anyone know how to adjust the volume? I would like to turn the volume down a little because it is too loud.
Thank you

**Comment by harbaum** [comment]
There's a poti on the amplifier board.You can use it with a small screwdriver to turn the volume down.

**Reply by gnokketto** [reply]
use display esp32.. Sure there's something to adjust ?, I'll check later

**Comment by harbaum** [comment]
What is "display esp32"? You need to be more specific!

**Comment by harbaum** [comment]
The yellow board? Read this:

https://github.com/hexeguitar/ESP32_TFT_PIO?tab=readme-ov-file#2-audio-amp-gain-mod

**Reply by gnokketto** [reply]
thanks
